### ISSUE TITLE
How to set it up?

### ISSUE THREAD

**Issue by 10june20** [Meowmaritus/DSAnimStudio#8]
I've tried searching online for a guide on how to set this up (for Dark Souls 3, specifically) but couldn't find anything. 

As you've probably guessed I have no idea what I'm doing, but I eventually figured out the .bdt and .bhd data files had to be decrypted(?) and did so, but I'm still not sure what DS Anim Studio hooks onto out of these? Or am I looking in entirely the wrong place?

**Comment by 10june20** [comment]
Bruh...

**Comment by AntonioCS** [comment]
I don't know much about how to use this program either but you can take a look at this guy: https://www.youtube.com/watch?v=NgvAbF8vcBA

He seems to know what he's doing. Good luck!

**Comment by 10june20** [comment]
@AntonioCS thank you, I'll have a look.

Full disclosure, I ended up here since I'm interested in messing around with Dark Souls 3 movesets

I used to do this in DS2 via CE [e.g](https://www.youtube.com/watch?v=um7VWPmvKwY) but it was purely for recording purposes, not something usable. I didn't mind that and, since my laptop is too old to run DS3 properly anyway, would be fine so long as I can replicate that kind of setup in DS Anim Studio

Problem is I can't figure out how to get it up and running?
